index.html

```diff
@@ -16,8 +16,14 @@
     communities are private, so you'll need to request an invite to join.  The individual communities
     are responsible for managing the invitation and vetting process.
   </p>
-  <ul>{% for community in site.data.communities %}
-    <li><a href="{{ community.url }}">{{ community.name }}</a></li>{% endfor %}
+  <ul>
+    {% for community in site.data.communities | sort 'name' %}
```

Comment:
It should, I messed up and forgot a ':' ... how are you running to test?  All I know how to do is push to github and let github format the pages?